docs/translations/api-docs/sign-in-page/sign-in-page.json

```diff
@@ -18,10 +18,10 @@
   },
   "classDescriptions": {},
   "slotDescriptions": {
+    "checkbox": "A custom checkbox placed in the credentials form",
```

Comment:
If we change the name as I proposed we'd have to update the description to something more generic too.